pkg/project/project.go

```diff
@@ -50,7 +49,7 @@ func Delete(client *occlient.Client, projectName string) error {
 	return nil
 }
 
-func DescribeProjects(client *occlient.Client) (ProjectList, error) {
+func List(client *occlient.Client) (ProjectList, error) {
```

Comment:
Can we please add comments for the function?

Reply:
I can see comments now so assuming this as resolved